python/lsst/eotest/image_utils.py

```diff
@@ -212,6 +212,12 @@ def bias_image(im, overscan, dxmin=5, dxmax=2, statistic=np.mean, bias_method='r
     Returns:
         An image with size equal to the input image containing the offset level.
     """
+    if bias_method not in ['mean', 'row', 'func', 'spline', 'none']:
+        raise RuntimeError('Bias method must be either "none", "mean", "row", "func" or "spline".')  
+
+    def dummy_none(im, overscan, dxmin, dxmax, **kwargs):
```

Comment:
Why do we need a "none" option?  Also this code block looks like it should be replacing lines 221-223 below.   That those lines are still there and everything works indicates that a "none" option isn't needed.

Reply:
I put it in b/c when we pass things to the batch farm all the options get turned into a string, and so we end up having stuff like --bias none or --bias spline at the command line, and that gets passed along to this function.   I could catch that further upstream if you prefer, but it was easy to catch it here.

Reply:
I marked this as resolved by mistake.  If we are going to keep this test here, then we should remove lines 221-223 below, since they redefine `method`.  Also one could define `method` first and test for membership in the dict, i.e.,
```
    def dummy_none(im, overscan, dxmin, dxmax, **kwargs):
        return 0.0
    method = {'mean' : bias, 'row' : bias_row, 'func' : bias_func, 'spline' : bias_spline, 'none' : dummy_none}

    if bias_method not in method:
        raise RuntimeError('Bias method must be either "none", "mean", "row", "func" or "spline".')  
```